CHANGELOG.md

```diff
@@ -232,14 +237,14 @@ ownership to contribute it to this project under the licence terms.
 - Merged Austin Ziegler's color-tools library (previously part of the Ruby PDF
   Tools project) with Matt Lyon's color library.
 
-  - The HSL implementation from the Color class has been merged into
+    - The HSL implementation from the Color class has been merged into
```

Comment:
```suggestion
  - The HSL implementation from the Color class has been merged into
```

It looks like your Markdown formatter has incorrectly decided that inner bullet points need to be indented further than they should. `deno fmt` is my preferred formatter for Markdown (it mostly does what I would do).